osquery/utils/aws_util.cpp

```diff
@@ -368,14 +340,13 @@ void getInstanceIDAndRegion(std::string& instance_id, std::string& region) {
     }
 
     initAwsSdk();
-    http::client::request req(kEc2MetadataUrl +
-                              "dynamic/instance-identity/document");
-    http::client::options options;
+    http::Request req(kEc2MetadataUrl + "dynamic/instance-identity/document");
+    http::Client::Options options;
     options.timeout(3);
-    http::client client(options);
+    http::Client client(options);
 
     try {
-      http::client::response res = client.get(req);
+      http::Response res = client.get(req);
       if (res.status() == 200) {
```

Comment:
As before, I think you need to check accept all 2xx values, like this:
```
if(beast_http::to_status_class(res.result()) != beast_http::status_class::successful)
```
And remove the member function `Response::status`